assets/src/apps/delivery/store/features/groups/actions/sequence.ts

```diff
@@ -140,7 +140,9 @@ export const getSequenceLineage = (
 };
 
 export const getSequenceInstance = () => {
```

Comment:
This would have been fine if we had named this useSequenceInstance, so just ignoring the errors for now.